src/bitcoin-tx.cpp

```diff
@@ -767,9 +767,7 @@ static std::string readStdin()
     if (ferror(stdin))
         throw std::runtime_error("error reading stdin");
 
-    boost::algorithm::trim_right(ret);
-
-    return ret;
+    return TrimString(ret);
```

Comment:
But this SHOULD be locale-tolerant!

Reply:
Why?

Reply:
It's trimming actual user input.

Reply:
I think we only want the Qt GUI to be localized; not RPC and command-line tools.

Reply:
Agree fully with sipa.

I've discussed localization extensively with various people in the project and IIRC I've never seen anyone except Luke make the claim that something other than Qt GUI should be localized.

Luke, is that your perception as well that you're representing a non-consensus opinion with regards to non-Qt GUI localization?

To be clear: having a contrarian opinion is obviously perfectly fine -- I love contrarians :)